Week-1-new/exercise1-2.html

```diff
@@ -1,91 +1,91 @@
-                    <p style="margin: 1% 6% 1% 6%;"><i class="la la-plus"></i>Add to</p>
-                    <p style="margin: 1% 6% 1% 6%;">...More</p>
-                </div>
-            </div>
-        </div>
-    </div>
-
-
-    <div>
-        <div style="display: flex;">
-            <p style="margin: 1% 6% 1% 6%;">1.One Dance</p>
-            <p style="margin: 1% 6% 1% 6%;">Crake feat Catkid & Cyla</p>
-            <p style="margin: 1% 6% 1% 6%;">2:54</p>
-            <p style="margin: 1% 6% 1% 6%;">CATTY CLOUD SYNC</p>
-        </div>
-        <div style="display: flex; background-color: rgb(221, 217, 217); border-radius: 5px;">
-            <p style="margin: 1% 6% 1% 6%;">1.One Dance</p>
-            <p style="margin: 1% 6% 1% 6%;">Crake feat Catkid & Cyla</p>
-            <p style="margin: 1% 6% 1% 6%;">2:54</p>
-            <p style="margin: 1% 6% 1% 6%;">CATTY CLOUD SYNC</p>
-        </div>
-        <div style="display: flex;">
-            <p style="margin: 1% 6% 1% 6%;">1.One Dance</p>
-            <p style="margin: 1% 6% 1% 6%;">Crake feat Catkid & Cyla</p>
-            <p style="margin: 1% 6% 1% 6%;">2:54</p>
-            <p style="margin: 1% 6% 1% 6%;">CATTY CLOUD SYNC</p>
-        </div>
-        <div style="display: flex; background-color: rgb(221, 217, 217); border-radius: 5px;">
-            <p style="margin: 1% 6% 1% 6%;">1.One Dance</p>
-            <p style="margin: 1% 6% 1% 6%;">Crake feat Catkid & Cyla</p>
-            <p style="margin: 1% 6% 1% 6%;">2:54</p>
-            <p style="margin: 1% 6% 1% 6%;">CATTY CLOUD SYNC</p>
-        </div>
-    </div>
-</div>
-</div>
-    <div style="display: flex; background-color: blue;">
-        <div style="margin: 1%;"><img style="margin: 1%; width:150px; height:150px;" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg" alt="cattyBoard"></div>
-        <div style="width: 100%;">
-            <div><p>If it ain't love</p></div>
-            <div style="display: flex; margin-top: 7%;">
-                <p style="margin: 1% 2% 2% 0%;">Catson Derulo</p>
-                <p style="margin: 1% 6% 1% 6%;"><i class="la la-step-backward"></i>Catson</p>
-                <p style="margin: 1% 6% 1% 6%;"><i class="la-ellipsis-h"></i>Catson</p>
-                <p style="margin: 1% 6% 1% 6%;"><i class="la la-step-forward"></i>Catson</p>
-                <p style="margin: 1% 6% 1% 6%;"><i class="la la-bars"></i>Catson</p>
-                <p style="margin: 1% 6% 1% 6%;"><i class="la la-random"></i>Catson</p>
-            </div>
-        </div>
-    </div>
-</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise1.2</title>
+    <link rel="stylesheet" href="./Line Awesome/css/line-awesome.min.css">
+    <style>
```

Comment:
write css in seperate file, try to avoid inline and internal css as much as possible